deploy/configure-assets-after-deployment.ts

```diff
@@ -91,10 +118,34 @@ export const deploy: DeployFunction = async (environment) => {
     await txn.wait(CONFIRMATIONS);
     hre.trace('Set market and certificate addresses in Removal');
   }
+
+  if (
+    (await market.getPurchasingTokenAddress()) !== purchaseTokenAddress ||
+    (await market.getPriceMultiple()) !== priceMultiple
+  ) {
+    txn = await market.setPurchasingTokenAndPriceMultiple(
+      purchaseTokenAddress,
+      priceMultiple
+    );
+    await txn.wait(CONFIRMATIONS);
+    hre.trace('Set USDC as purchase token with price multiple of 2000');
+  }
+
+  if ((await market.getNoriFeePercentage()) !== BigNumber.from(25)) {
+    txn = await market.setNoriFeePercentage(25);
+    await txn.wait(CONFIRMATIONS);
+    hre.trace('Set fee percentage to 25');
```

Comment:
I wonder if we should define this fee percentage somewhere more central, like at least as a global variable in this script but maybe as an environment variable or something

Reply:
this might also be helpful for the price multiple values used above

Reply:
agreed, we're kindof lacking an environment to contract config mapping somewhere more organized.